### DISCUSSION TITLE
Updated Workaround for “Google couldn't confirm this attempt to sign in is safe. If you think this is a mistake, you can close and try again to sign in” warning

### DISCUSSION THREAD

**Discussion by mrbonkerz** [qnblackcat/uYouPlus#904] [General]
Issue has been fixed in [v18.08.1-2.3.1](https://github.com/qnblackcat/uYouPlus/releases/tag/v18.08.1-2.3.1)

~~[uYou Login V2.1 Video Guide](https://imgur.com/a/KpgRtE4)~~

~~1) download this iOS Shortcut -> [uYou Login V2.2](https://www.icloud.com/shortcuts/214c4403becb4c25911471a3b84a1c42)~~
~~2) download Vanilla YouTube~~
~~3) enter email address on both Vanilla and Cracked YouTube. Proceed to the password page without logging in~~
~~4) **instead of the password page, it will now be "couldn't sign you in" page**. Copy login URL from the title (long press on title) of the Cracked YouTube first~~
~~5) run shortcut and paste URL when prompted; 10 seconds delay starts (to execute the next step)~~
~~6) switch to Vanilla YouTube, copy login URL and paste URL when prompted~~
~~7) sign in as per normal using the uYou Login in-app browser~~
~~8) if nothing happens after clicking "open", swipe up/home button~~

~~If it doesn't work:~~
~~1) sign out from all Google account~~
~~2) do not need to login through the Vanilla YouTube~~
~~3) copy the login URL by holding onto the title in the YouTube App~~
~~4) keep both Vanilla and Cracked YouTube App open when running the uYou Login shortcut~~
~~5) if the shortcut opens the Vanilla YouTube, try removing the Vanilla YouTube first before logging in on the uYou Login shortcut~~

~~References: 
~~Vanilla/Regular YouTube = official YouTube from AppStore~~
~~Cracked YouTube = uYou Plus YouTube~~

~~Troubleshooting:
~~1) Screen record your issue and upload to YouTube~~
~~2) Post your Cracked & Vanilla URL here~~

~~Changes:~~
~~9 Feb - shortcut updated to v2.2; allowing the use of "couldn't sign you in" page link instead of the password page~~

-mrbonkerz

**Comment by ghost** [comment]
Hi,

**EDIT: It's working perfectly I should've updated the app before.**

Thank you 

---------------------------------------------
Step 3 is not reproductible 

“ 3. enter email address on both Vanilla and Cracked YouTube. Proceed to the password page without logging in”

Doesn't work since I can't get to the password page on the uYou version.

I may have missed something.

**Reply by cukingpro** [reply]
@RAtIG  I got same problem. How did you solve it?

**Reply by ghost** [reply]
I updated the app

**Comment by Rosssolimo** [comment]
This works as of today on iPad Pro 2018, iPadOS 16.2, thank you!

**Comment by RoToN4iK** [comment]
Perfectly works on iPhone 13, iOS 16.1.2, ty <3

**Comment by Sebastin3244** [comment]
I downloaded the shortcut on my ipad.  I couldn't find a URL in the title, because, I am blind, and I use voice over.  Is their another accessible option for me?  How can I contact google and ask them to stop blocking important third party apps.  They save our money.  We don't have to pay for youtube premium.

**Comment by KOLIJOH** [comment]
App no save iPhone 7 Plus I follow all steps watch video but not work for my so I previously try sign in on previous method this work but i delete my uyouplus(scarlet certificate problems) and try again and this didn’t work anymore no safari opens no clipboard link but 2.1 fix no clipboard link for my but I received error who this no save attempt or app I try open link in safari and manually sign in uyouplus after command but ye error (sorry for English)

**Comment by ulcins** [comment]
Works on iPhone 12 mini, iOS 16.2, thank you so much!

**Comment by trancaochiluan** [comment]
works for me, really work, thanks a lottttttt

**Comment by shaqqoz** [comment]
How are y’all copying the url from the YouTube sign in page? 
![E678777B-C68A-45D8-B8FF-800659EB532E](https://user-images.githubusercontent.com/117412455/213926968-118ae80b-10dd-48f2-86ce-9c2d7b07c2d9.jpeg)

**Reply by mrbonkerz** [reply]
long press on the "accounts.google.com"

**Comment by Knoxyzz** [comment]
worked for me (Iphone 11 IOS 16.0) tysm

**Comment by yukioktk** [comment]
THANK YOUUUUUU

Works on Iphone SE 2nd gen.  ios 16.2

**Comment by Sebastin3244** [comment]
I just run the shortcut and type in accounts.google.com.

**Comment by mr-utilman** [comment]
It doesnt work on iphone 14 pro Max iOS 16.2, because:

the new updated cracked app just crashes on iOS 16.xx devices so i use one older like YouTube_16.42.3_uYou_2.1.ipa

when i use the shortcut nothing is working. Allways saying that Google couldnt ... blabla... would be nice to have some working workaround. Also tried to delete everything after the "="

when is coming a working solution for iOS 12.xx devices? 

Thanks

**Comment by Sebastin3244** [comment]
Is the problem fixed in uyou+ 18?

**Comment by asamahy** [comment]
works with youtube v17.36.4 and the uyou version from that time as well.

**Reply by Sebastin3244** [reply]
Is their another way that I could log in to uyou+?

**Comment by oleoth** [comment]
On uYou+ v18.01.6-2.3.1, Youtube 18.03.3 and iPadOS 16.3 this just crashes uYou+ after pressing "Open" on step 8, and does not successfully sign in.

**Comment by Ned-Flanders** [comment]
Worked perfectly for me, I had to close real youtube though before running the shortcut after copying the URL.

 Iphone 14 Pro Max IOS 16.3, uYou v18.01.6-2.3.1, YouTube 18.04.3

Thanks a lot for this, you saved me a headache

**Reply by mr-utilman** [reply]
Doesnt work on iPhone 14 pro max

**Reply by Ned-Flanders** [reply]
It worked on my pro max, the model of the device shouldn’t matter anyway as it’s related to a third-party app. There aren’t different builds of the app for different devices so if it works on one it will work on all of them.

**Reply by mr-utilman** [reply]
That's not so easy as you think; it might be one build that's correct but to say it should work on all of them isn't the right answer - just look above ☝️ all the comments and much more through other issue site's. It might also have a lot of different reasons on the user's configuration site and so on...

**Reply by Ned-Flanders** [reply]
Those things aren’t related to the device though. You said it doesn’t work on a 14 pro max and then started saying about user configuration and issues with the site. I was saying that if a specific build/user configuration works on one device it will work on them as nothing changes across devices. It’s related to a third-party app and manipulates the auth token. That doesn’t have anything to do with the device.

**Reply by nosar77** [reply]
I had the same issue. I got mine to work, I uninstalled the original YouTube before I inputted the url from the original YouTube and my login worked

> That's not so easy as you think; it might be one build that's correct but to say it should work on all of them isn't the right answer - just look above ☝️ all the comments and much more through other issue site's. It might also have a lot of different reasons on the user's configuration site and so on...

**Comment by dirtfemme** [comment]
Just writing to comment that this worked for me today on 16.1 on 14 Pro (not JB obvi). Thank you!

**Comment by mgcrea** [comment]
In my case it always re-opens the vanilla youtube app even if I close it just before the final login.

**Reply by Ned-Flanders** [reply]
That happened to me, I had to close the vanilla YouTube app after copying the URL but keep the cracked one open.

**Comment by HugoMskn** [comment]
couldnt get it working on my iphone 14 with ios 16.3

**Reply by mr-utilman** [reply]
I also not on my iphone cause the app crashes after launch

**Comment by Sebastin3244** [comment]
is their a faster way?  the shortcut doesn't work for me.  Google is being mean.  How can I contact google and ask them to allow people with third party youtube apps to use google accounts?

**Reply by HugoMskn** [reply]
There is, add me on discord I'll explain Hugo ;)#3737

**Reply by mr-utilman** [reply]
can you help me as well?

**Reply by HugoMskn** [reply]
Yeh add me

**Reply by mr-utilman** [reply]
how, whats the username

**Reply by HugoMskn** [reply]
Hugo ;)#3737
That's my username

**Reply by Sebastin3244** [reply]
what is the link to your discord server

**Reply by Sebastin3244** [reply]
> Hugo ;)#3737
> 
> That's my username 

what is the link to your discord server

**Reply by HugoMskn** [reply]
I don't have a discord server

**Reply by Sebastin3244** [reply]
> I don't have a discord server 

what is the faster way

**Reply by Sebastin3244** [reply]
> I don't have a discord server 

how do i add you to discord

**Reply by Sebastin3244** [reply]
What is the faster way of logging into uyou+?

**Reply by HugoMskn** [reply]
Mine as its just before installing or you can dm on Twitter

**Reply by Sebastin3244** [reply]
What is the faster way of logging in to uyou+?  I uninstalled the dirty original youtube, typed accounts.google.com in to the shortcuts app, and I still get the error.

**Comment by huutanphp** [comment]
Works fine on iphone 13 ios 15.6RC.
uYouPlus_18.01.6_2.3.1 and youtube 18.04.3
I sideloaded Uyou with apple developer account and they are actually logged into Uyou

**Reply by Sebastin3244** [reply]
How do I do that?

**Reply by huutanphp** [reply]
I've done them many times and it worked when I tried again in the morning 👍

**Comment by FilipLupaescu** [comment]
![photo_2023-02-06_02-02-44](https://user-images.githubusercontent.com/65253848/216858919-22dd7209-c704-4d8f-b91e-27e161c06202.jpg)
There is no link to copy in the vanilla app...

**Reply by Ned-Flanders** [reply]
You need to put your email in first

**Reply by Sebastin3244** [reply]
What is a faster way that I can use to log in to uyou+?

**Reply by Ned-Flanders** [reply]
> What is a faster way that I can use to log in to uyou+?

You need to use this shortcut, watch the video if you’re struggling

**Reply by Sebastin3244** [reply]
What is the link to the video?  How do I get the shortcut?  I am not use to that.  I am always use to just logging in, and watching videos.  google is being babyish.  Google is being mean. The modded youtube apps are spectacular.

**Reply by Ned-Flanders** [reply]
> What is the link to the video? How do I get the shortcut? I am not use to that. I am always use to just logging in, and watching videos. google is being babyish. Google is being mean. The modded youtube apps are spectacular.

Video link: 

https://imgur.com/a/KpgRtE4

Shortcut link:

https://www.icloud.com/shortcuts/93327cf53fdd4ca0a6f913dbd4c859df

**Reply by Sebastin3244** [reply]
the video is not telling me how to do it

**Reply by Ned-Flanders** [reply]
> the video is not telling me how to do it

If you follow the video step by step it will work. Which part are you struggling with?

**Reply by Sebastin3244** [reply]
The video is not telling me anything.

**Reply by Sebastin3244** [reply]
I got uyou+ working.  Thank you for your help.

**Reply by huyle0406** [reply]
When sideloading with Bullfrog Assistant, I cannot get to the screen where I input my password. It show error after I put my email addres in. Can anyone help me solve this?

**Reply by Ned-Flanders** [reply]
> When sideloading with Bullfrog Assistant, I cannot get to the screen where I input my password. It show error after I put my email addres in. Can anyone help me solve this?

I’ve never used bullfrog so I wouldn’t be able to assist, I’d recommend AltStore or troll store if possible

**Reply by darksniper-2k47** [reply]
HEY I FOLLOWED THE VIDEO BUT I GOT AN ERROR 

PLEASE TELL ME WHAT I DID WRONG

https://youtu.be/pvI1QXRcXzo

**Reply by sirlupusdev** [reply]
@darksniper-2k47 You will need to proceed to the Input password page in the cracked YT app and input that link into the shortcut. You can also use that link even if it states "Can't sign you in, because Google can't confirm it's safe" (Google does that know when proceeding to the passwd page).

**Reply by mrbonkerz** [reply]
@darksniper-2k47 use the updated v2.2 shortcut

**Reply by yodaluca23** [reply]
> I’ve never used bullfrog so I wouldn’t be able to assist, I’d recommend AltStore or troll store if possible 

[SideStore](https://github.com/SideStore/SideStore) over altstore...

**Comment by huyle0406** [comment]
When sideloading with Bullfrog Assistant, I cannot get to the screen where I input my password. It show error after I put my email addres in. Can anyone help me solve this

**Reply by sirlupusdev** [reply]
I'm also having this issue when sideloading with AltStore, Google seems to try to break it again.

**Reply by sirlupusdev** [reply]
Nevermind, I had this issue too. I solved the problem by copying the youtube url of cracked YT **before** typing in the mail, pasting it into Safari, then typing in the email (but not the password). After that, proceed with the instructions in the initial message (treating the safari URl as cracked YT URL). You may need to swipe up/press the home button if the opened tab doesn't close. It worked for me on the third attempt, so you may need to retry.

**Reply by sirlupusdev** [reply]
Additionally, I had to close the original app after copying the URL, just fyi.

**Reply by ryan-fung** [reply]
> Nevermind, I had this issue too. I solved the problem by copying the youtube url of cracked YT **before** typing in the mail, pasting it into Safari, then typing in the email (but not the password). After that, proceed with the instructions in the initial message (treating the safari URl as cracked YT URL). You may need to swipe up/press the home button if the opened tab doesn't close. It worked for me on the third attempt, so you may need to retry.

Even after copying the URL before entering the email, and putting it in Safari, then entering my email in Safari gives me a "Google couldn't confirm this attempt to sign-in is safe" error unfortunately

**Reply by huyle0406** [reply]
> Nevermind, I had this issue too. I solved the problem by copying the youtube url of cracked YT **before** typing in the mail, pasting it into Safari, then typing in the email (but not the password). After that, proceed with the instructions in the initial message (treating the safari URl as cracked YT URL). You may need to swipe up/press the home button if the opened tab doesn't close. It worked for me on the third attempt, so you may need to retry.

I tried and no luck. It is normal if sideloading with altstore or sidestore on same apple id. But altstore has to connect to pc and sidestore ussually lock my account so bullfrog is the most convenient way

**Reply by mrbonkerz** [reply]
Please try the new updated v2.2 shortcut

**Comment by quixotic120** [comment]
Getting an “invalid url” error from the vanilla link. Complaining about the &TL=xxxx part of the vanilla url

**Reply by quixotic120** [reply]
Also can’t get to the password screen in the cracked app fwiw, just get the “google won’t let you sign in bc this method isn’t trusted” error page

**Reply by mrbonkerz** [reply]
Please try the new updated v2.2 shortcut

**Reply by quixotic120** [reply]
Worked perfectly on first try, thanks!

**Comment by mrbonkerz** [comment]
Shortcut has been updated to v2.2 to fix "couldn't sign you in" password page

**Reply by sirlupusdev** [reply]
Does this mean I can use the "Couldn't sign you in" page in the cracked YT as the cracked URL or is there something else that I'd have to do?

**Reply by mrbonkerz** [reply]
Yes. You can use the error page from the cracked YouTube app.

**Reply by mgcrea** [reply]
@mrbonkerz tried again today but looks like they now directly block after the login/email form, can't navigate to the password page anymore and get "couldn't sign you in" instead of the password prompt.

**Reply by mrbonkerz** [reply]
You can copy the link of the "couldn't sign you in" page from the cracked app. It should still work.

**Reply by mgcrea** [reply]
@mrbonkerz unfortunately still not working, when logging in with the patched URL I get an infinite spinner (but the login seems successful) and get redirected (or stays) in the vanilla app. If I swipe to go back to the cracked app I get back to the "couldn't sign you in page" I had before. Both apps stay unlogged:

Here is my URLs: one thing that stands out is the `device_name` that looks wrong in the cracked app. not sure if it changes anything.

```
Cracked:
https://accounts.google.com/embedded/setup/v2/safarivc/deniedsigninrejected?service=youtube&ltmpl=youtube_main&as=$AS_CRACKED&scope=$SCOPE&client_id=$CLIENT_ID&redirect_uri=$REDIRECT_URI&sarp=1&delegated_client_id=$DELEGATED_CLIENT_ID&hl=$HL&device_name=iPhone&auth_extension=$CRACKED_AUTH_EXTENSION&code_challenge_method=S256&code_challenge=$CRACKED_CODE_CHALLENGE&svc_ac=1&sfas=1&flowName=GlifSetupSafariVC&TL=$CRACKED_TL

Vanilla:
https://accounts.google.com/signin/v2/challenge/pwd?service=youtube&ltmpl=youtube_main&as=$AS_VANILLA&scope=$SCOPE&client_id=$CLIENT_ID&redirect_uri=$REDIRECT_URI&sarp=1&delegated_client_id=$DELEGATED_CLIENT_ID&hl=$HL&device_name=iPhone%20d’FooBar&auth_extension=$VANILLA_AUTH_EXTENSION&system_version=16.3&app_version=18.05.2&kdlc=1&kss=uca&lib_ver=3.4&device_model=iPhone14%2C3&code_challenge_method=S256&code_challenge=$VANILLA_CODE_CHALLENGE&svc_ac=1&sfas=1&flowName=GlifSetupSafariVC&cid=1&navigationDirection=forward&TL=$VANILLA_TL

Patched:
https://accounts.google.com/signin/v2/challenge/pwd?service=youtube&ltmpl=youtube_main&as=$AS_CRACKED&scope=$SCOPE&client_id=$CLIENT_ID&redirect_uri=$REDIRECT_URI&sarp=1&delegated_client_id=$DELEGATED_CLIENT_ID&hl=$HL&device_name=iPhone&auth_extension=$CRACKED_AUTH_EXTENSION&code_challenge_method=S256&code_challenge=$CRACKED_CODE_CHALLENGE&svc_ac=1&sfas=1&flowName=GlifSetupSafariVC&TL=$VANILLA_TL
```

**Reply by sirlupusdev** [reply]
@mgcrea Try to copy the vanilla URL and then force-close the vanilla app. If you get an infinite loading screen/spinner/bar, just swipe up or press your home button, I had that issue as well - and after pressing on "Open in YT" when it prompted me, it seemed to open that in the background.

**Reply by mgcrea** [reply]
@lupus0802 should the cracked app be the one to be opened after the login? In my case it restarts the vanilla app.

**Reply by sirlupusdev** [reply]
Yes, the cracked app would need the one to be opened. That behaviour is quite interesting tho. I'm not sure if that occurs in because of the Shortcut update or something else.

**Reply by filandre1** [reply]
Here, only work when i use login url from Safari. 

The process its the same but without Vanilla installed and use the Safari login url.

**Reply by SupDos** [reply]
2.2 worked perfectly for me after a backup restore logging me out, Thank you!

**Comment by CrocodileDuty** [comment]
Working flawlessly

**Comment by iamfromreallife** [comment]
This works but there's an easier way: When the scripts asks for vanilla link, don't paste anything, just type accounts.google.com, login with your google/youtube account and uyou plus (or other tweaked youtube apps) will also be logged in after. No need to switch apps, copy, pasting, all that stuff.

**Comment by qStars** [comment]
If you enter an account name and click Next, it will not work.  You can go to iOS settings, Safari browser - "Advanced" - Experimental Features - RequestIdleCallback close and then log in

**Reply by sirlupusdev** [reply]
You can use the "Couldn't sign you in link" if that's what you're talking about.

**Reply by qStars** [reply]
> You can use the "Couldn't sign you in link" if that's what you're talking about.

I use Trollstore sideloaded software.  After setting as above.  I can then log in directly from the sideloaded in-app web page.  There are no other external operations.

**Comment by 123917** [comment]
An error occurred while extracting the channel list of the account.

**Comment by ademkarakaya** [comment]
Can confirm that this worked today 
(13-Feb-2023)

Region: 
AU

Phone:
iPhone 14 Pro Max

OS: 
iOS 16.2

uYouPlus version (.ipa file):
18.01.6_2.3.1

Sideload Method:
Altstore

The shortcut v2.2 is what enabled it to work. The first time I did it, the on-screen keyboard malfunctioned. 

The second time it worked and prompted to go back to 'YouTube' (i.e uYouPlus) but it just hung on a spinning load icon, so I had to swipe up on the (home bar?) to "exit" the shortcut and found myself in the app. 

Thank you @mrbonkerz for spending the time to develop the shortcut and for the solution.

**Comment by ukashah1** [comment]
![image](https://user-images.githubusercontent.com/77582810/218316749-92d6ba84-abc6-4755-a87b-ef310109c79d.jpeg)

what's wrong?

**Reply by sirlupusdev** [reply]
You can also use the URL even if this screen shows. Google implemented a check so you technically can't sign in, however, if you use the workaround described in the first message, it still works.

**Reply by ukashah1** [reply]
> You can also use the URL even if this screen shows. Google implemented a check so you technically can't sign in, however, if you use the workaround described in the first message, it still works.

thx. but i used the option "Sideload Spoofer" in Sideloadly and now i can log in flawlessly!

**Comment by Skauyy** [comment]
It’s work on my 6s plus

**Comment by FrostLight131** [comment]
As of Feb-13, seems like Google has patched it. There's no "enter your password" box, just simply "You can't sign in to this app because Google can't confirm that it's safe". No workarounds have worked, not even the shortcut.

When I used the shortcut, after pasting both cracked and vanilla youtube url, after entering my email it just says "Google couldn't confirm this attempt to sign in is safe" without prompting me to enter password.

**Reply by ukashah1** [reply]
There's a workaround. Use Sideloadly and insert ipa file then below the "Advanced Options" tick the "Inject dylibs/frameworks" after that, tick "Sideload Spoofer" and boom, can login like normal. This also logs you in YT Music Ultimate after the login session in uYou+.

**Reply by FrostLight131** [reply]
> There's a workaround. Use Sideloadly and insert ipa file then below the "Advanced Options" tick the "Inject dylibs/frameworks" after that, tick "Sideload Spoofer" and boom, can login like normal. This also logs you in YT Music Ultimate after the login session in uYou+.

this man deserves nobel peace prize. Works 

So how do i "refresh" the cracked app? Do I have to reinstall it every 7 days or is it like altstore where i can just click a button and refresh

**Reply by sirlupusdev** [reply]
@FrostLight131 You will have to sideload it manually every 7 days as Sideloadly is still working on an auto refresh. 
If you sideload it with AltStore/SideStore, you can also use the "Google can't sign you in" page as the cracked YT Url as long as you're using version 2.2 of the shortcut.

**Reply by ukashah1** [reply]
> > There's a workaround. Use Sideloadly and insert ipa file then below the "Advanced Options" tick the "Inject dylibs/frameworks" after that, tick "Sideload Spoofer" and boom, can login like normal. This also logs you in YT Music Ultimate after the login session in uYou+.
> 
> 
> 
> this man deserves nobel peace prize. Works 
> 
> 
> 
> So how do i "refresh" the cracked app? Do I have to reinstall it every 7 days or is it like altstore where i can just click a button and refresh

If you want to refresh using AltStore, just uninstall uYou+ and install it again on AltStore. Your account will be automatically logged in.

**Comment by binnichtaktiv** [comment]
> [uYou Login V2.1 Video Guide](https://imgur.com/a/KpgRtE4)
> 
> 1. download this iOS Shortcut -> [uYou Login V2.2](https://www.icloud.com/shortcuts/214c4403becb4c25911471a3b84a1c42)
> 2. download Vanilla YouTube
> 3. enter email address on both Vanilla and Cracked YouTube. Proceed to the password page without logging in
> 4. **instead of the password page, it will now be "couldn't sign you in" page**. Copy login URL from the title (long press on title) of the Cracked YouTube first
> 5. run shortcut and paste URL when prompted; 10 seconds delay starts (to execute the next step)
> 6. switch to Vanilla YouTube, copy login URL and paste URL when prompted
> 7. sign in as per normal using the uYou Login in-app browser
> 8. if nothing happens after clicking "open", swipe up/home button
> 
> If it doesn't work:
> 
> 1. sign out from all Google account
> 2. do not need to login through the Vanilla YouTube
> 3. copy the login URL by holding onto the title in the YouTube App
> 4. keep both Vanilla and Cracked YouTube App open when running the uYou Login shortcut
> 5. if the shortcut opens the Vanilla YouTube, try removing the Vanilla YouTube first before logging in on the uYou Login shortcut
> 
> References: Vanilla/Regular YouTube = official YouTube from AppStore Cracked YouTube = uYou Plus YouTube
> 
> Troubleshooting:
> 
> 1. Screen record your issue and upload to YouTube
> 2. Post your Cracked & Vanilla URL here
> 
> Changes: 9 Feb - shortcut updated to v2.2; allowing the use of "couldn't sign you in" page link instead of the password page
> 
> -mrbonkerz

Version 2 didn’t work but 2.2 did thank you very much 🙏🏻

**Reply by harbour0** [reply]
Can confirm version 2.2 worked for me as of 15/02/23

**Comment by callmesupls** [comment]
> [uYou Đăng nhập v2.1 Video Hướng dẫn](https://imgur.com/a/KpgRtE4)
> 
> 1. tải xuống Phím tắt iOS này -> [uYou Đăng nhập V2.2](https://www.icloud.com/shortcuts/214c4403becb4c25911471a3b84a1c42)
> 2. tải xuống Vanilla YouTube
> 3. nhập địa chỉ email trên cả Vanilla và Cracked YouTube. Tiếp tục đến trang mật khẩu mà không cần đăng nhập
> 4. **Thay vì trang mật khẩu, bây giờ nó sẽ là trang "Không thể đăng nhập cho bạn".** Sao chép URL đăng nhập từ tiêu đề (nhấn và giữ tiêu đề) của YouTube đã bẻ khóa đầu tiên
> 5. chạy phím tắt và dán URL khi được nhắc; Độ trễ 10 giây bắt đầu (để thực hiện bước tiếp theo)
> 6. chuyển sang Vanilla YouTube, sao chép URL đăng nhập và dán URL khi được nhắc
> 7. đăng nhập như bình thường bằng trình duyệt trong ứng dụng Đăng nhập uYou
> 8. Nếu không có gì xảy ra sau khi nhấp vào "Mở", hãy vuốt lên / nút Trang chủ
> 
> Nếu nó không hoạt động:
> 
> 1. đăng xuất khỏi tất cả tài khoản Google
> 2. không cần đăng nhập thông qua Vanilla YouTube
> 3. sao chép URL đăng nhập bằng cách giữ vào tiêu đề trong Ứng dụng YouTube
> 4. luôn mở cả Ứng dụng YouTube Vanilla và Cracked khi chạy phím tắt Đăng nhập uYou
> 5. nếu phím tắt mở Vanilla YouTube, trước tiên hãy thử xóa Vanilla YouTube trước khi đăng nhập vào phím tắt Đăng nhập uYou
> 
> Tham khảo: Vanilla/Regular YouTube = official YouTube from AppStore Cracked YouTube = uYou Plus YouTube
> 
> Troubleshooting:
> 
> 1. Ghi lại sự cố màn hình của bạn và tải lên YouTube
> 2. Đăng URL Cracked & Vanilla của bạn tại đây
> 
> Thay đổi: 9 Feb - phím tắt được cập nhật lên v2.2; Cho phép sử dụng liên kết trang "Không thể đăng nhập bạn" thay vì trang mật khẩu
> 
> -Mrbonkerz

iPhone 12 Pro Max, ios 16.1.1, it worked, thank you

**Comment by laliin** [comment]
Anyone knows what I’m doing wrong? Help pls

Model: iPhone xr
OS: 16.3.1
IPA: uYouPlus_18.01.6_2.3.1

https://youtu.be/mk7gPP-rBEE

**Reply by callmesupls** [reply]
Please click done, and go to the account section to see it is logged in

**Comment by hungchannels123** [comment]
When I enter my password I got switch to Youtube Vanilla iPhone Xs 16.3.1 uYouPlus_18.01.6_2.3.1.ipa

**Comment by menotuu** [comment]
Hello
Iam able to login at the uYou Login V2.2 login password question, but it doesn't nothing do with one of the Youtube Apps.... Still unsigned there.


Solution 2 worked:
Removing vanilla Youtube before signing in!! Nice

**Reply by yodaluca23** [reply]
> Hello Iam able to login at the uYou Login V2.2 login password question, but it doesn't nothing do with one of the Youtube Apps.... Still unsigned there.

Try deleting the YoutTube app from the app store, after you copy its URL when the shortcut allocates it for copying the URL...

**Reply by hungchannels123** [reply]
> > Hello Iam able to login at the uYou Login V2.2 login password question, but it doesn't nothing do with one of the Youtube Apps.... Still unsigned there.
> 
> Try deleting the YoutTube app from the app store, after you copy its URL when the shortcut allocates it for copying the URL...

It okay for me, iPhone Xs 16.3.1 uYouPlus_18.01.6_2.3.1.ipa

**Comment by prashantcruise** [comment]
I found a workaround in Reddit I forgot the name (Thanks for this ).
Step 1:- Use _sideloadly_ install this with` inject dylibs and select sideload spoofer. `  
Step 2: Login to your account will work
Step 3: to get rid of the sidelodly notification again install it normal as you do. 
 
**
![image](https://user-images.githubusercontent.com/30691886/220698191-c30b2688-0756-401a-b679-e8d7ef80863c.png)
**

**Reply by Noviinha** [reply]
Just dusting off my github account to come say, this is the only method that worked for me. thanks

**Reply by asd071589** [reply]
This method definitely works the most, thank you :)

**Reply by InnocentChap** [reply]
This works for me, Iphone 13 version16.4.1., also the uYou+ I downloaded was the latest version available as of today, 6/12/23.

**Reply by Sbriny** [reply]
thx bro.

**Reply by ppamorim** [reply]
Not working anymore.

**Comment by mgcrea** [comment]
Anyone encounters "Error fetching channel list for account" once logged in, looks like it won't let me in even if logged in.

The login finally seemed to work once I deleted the YouTube app after copying its link (did open the cracked app).

**Comment by IvanovGeorge** [comment]
This works

**Comment by BlasterOverlord** [comment]
Seems like I'm having a completely different issue altogether.
For me, it says my browser has cookies blocked, hence I can't sign in? However I don't have cookies blocked. Plus I believe the browser that opens for signing in, is one provided by uyou+ not my browser.

**Comment by qnblackcat** [comment]
@mrbonkerz The issue has been fixed in [v18.08.1-2.3.1](https://github.com/qnblackcat/uYouPlus/releases/tag/v18.08.1-2.3.1) (or higher). Could you mention it in the post?

**Comment by Coreyifortune** [comment]
> in v18.08.

It doesn't let me login in a normal way. Still said google can't confirm it's safe. 😢

**Reply by qnblackcat** [reply]
Your sideload tool is?

**Reply by Coreyifortune** [reply]
> Your sideload tool is?

I am using uScarlet

**Reply by cukingpro** [reply]
I’m using Scarlet and the app does not show on home screen after installing. V18.08

**Reply by cukingpro** [reply]
Reinstall and now i cant login

**Reply by cukingpro** [reply]
Install with esign and can login now.

**Reply by yodaluca23** [reply]
> > Your sideload tool is?
> 
> I am using uScarlet

I'm pretty sure Scarlet automatically changes the bundle ID, this is likely your problem... (another reason why Scarlet is absolutely garbage!)

**Reply by Coreyifortune** [reply]
> > > Your sideload tool is?
> > 
> > I am using uScarlet
> 
> I'm pretty sure Scarlet automatically changes the bundle ID, this is likely your problem... (another reason why Scarlet is absolutely garbage!)

Any alternatives besides altstore? I used scarlet since it does not require me to use a desktop and weekly sideloading is no longer required.

**Reply by yodaluca23** [reply]
> > I'm pretty sure Scarlet automatically changes the bundle ID, this is likely your problem... (another reason why Scarlet is absolutely garbage!)
> 
> Any alternatives besides altstore? I used scarlet since it does not require me to use a desktop and weekly sideloading is no longer required.

[SideStore](https://github.com/SideStore/SideStore)

**Reply by mrbonkerz** [reply]
Try the latest version of uYou+

**Reply by yrmuq** [reply]
> > Your sideload tool is?
> 
> I am using uScarlet

uYouplus login worked well with Scarlet using Siri shortcut 2.2!  You just need to quicly delete original YouTube app after coping url from it, paste url in the box and entering password in the next step. It asking you to open YouTube app, just confirm it and all good)
uYouplus with Scarlet is the best free option!

**Comment by huutanphp** [comment]
I have set up the dylib file and I have injected it into ipa huyou.  wow amazing they work fine when login is not blocked by google

**Comment by Kayxue** [comment]
I followed all method without using computer, but still can’t worked.
It showed me “You can’t signed in to this app because google can’t confirm that it’s safe” when I logged in in uYou+, and after I saw that message and copied the link to safari, it said ……
![IMG_0160](https://user-images.githubusercontent.com/26020165/222319752-7bec1784-48df-4a1f-86af-058d0be2e146.jpeg)
Side loaded with scarlet

**Reply by zanavgustin** [reply]
U need to delete vanilla yt when u copy the url from it 
Vanilla=yt from appstore

**Comment by Chereyoma** [comment]
just installed, command 2.2, and all the instructions, after the link to the address, removed the vanilla YouTube and entered the application

**Comment by notvlexz** [comment]
Uyouplus has a new version and the workarounds aren’t working for me.
uYou 18.29.1_3.0.1

I’m entering the gmails on both youtube apps but not entering passwords, copying links, replacing the URL’s from official to tweak after &TL= and there’s no space, whenever I paste the new link in safari, I have to enter my gmail again, then after I can’t type in password.

**Reply by LaziChicken7** [reply]
do u have link uyouplus 18.29.01? thanks

**Comment by Hurusio** [comment]
I followed everything step-by-step but instead of getting to the login page on the cracked youtube I got sent to the recovery one, and if I try to recover the email it says that Google can't confirm that this attempt to sign in is safe. Any workarounds/solutions?

**Reply by surajgupta101** [reply]
Did you find any solution?

**Comment by notvlexz** [comment]
Hi,

no I didn’t. I just the a previous version :)

**Reply by surajgupta101** [reply]
Which version works for you ?

**Comment by notvlexz** [comment]
18.14.1

**Reply by surajgupta101** [reply]
Thanks

**Comment by clasenpickle** [comment]
I’m having this problem when I updated the app, never happened until now and the shortcut isn’t working even if I delete vanilla

**Reply by car20001** [reply]
If anyone is having issues with this in 2024. Just download an older version of the tweaked youtube app until you find a version that lets you sign in. Once your signed in download the newest version of the tweaked YouTube app and install it over the old version.(without deleting the app) Walla! Problem solved..... this works with AltStore, trollstore, sideloadly.

**Reply by brettw02** [reply]
What app version worked for you?

**Reply by car20001** [reply]
> If anyone is having issues with this in 2024. Just download an older version of the tweaked youtube app until you find a version that lets you sign in. Once your signed in download the newest version of the tweaked YouTube app and install it over the old version.(without deleting the app) Walla! Problem solved..... this works with AltStore, trollstore, sideloadly. 




uYouEnhanced_19.16.3-3.0.3.ipa

I had to install this version to be able to log in. And then update it to the newest version without deleting the old app

**Reply by brettw02** [reply]
Worked like a charm! Thank you so much

**Comment by car20001** [comment]
uYouEnhanced_19.16.3-3.0.3.ipaI had to install this version to be able to log in. And then update it to your newest version without deleting the app. On Sep 12, 2024, at 5:32 PM, Meridius_Decimus ***@***.***> wrote:﻿
What app version worked for you?
